src/server/actions/EventVolunteer.ts

```diff
@@ -0,0 +1,34 @@
+import { CheckInVolunteerRequest } from '@/types/dataModel/eventVolunteer';
```

Comment:
This whole file probably heavily conflicts with the main branch, so you will need to pull from main here and do that.